src/openzaak/components/documenten/tests/test_objectinformatieobject.py

```diff
@@ -36,8 +35,13 @@
     get_zaak_response,
     get_zaakinformatieobject_response,
 )
-from openzaak.tests.utils import mock_brc_oas_get, mock_zrc_oas_get
+from openzaak.tests.utils import mock_brc_oas_get, mock_vrc_oas_get, mock_zrc_oas_get
+from openzaak.tests.utils.verzoeken import (
+    get_verzoek_response,
+    get_verzoekinformatieobject_response,
+)
```

Comment:
perhaps this should go into `openzaak.contrib.verzoeken`?

In the future we can then move `openzaak.bag`, `openzaak.nlx`... in there as well.